tools/tag-release

```diff
@@ -230,8 +230,10 @@ def _build_parser():
 
     parser.add_argument(
         "action",
-        choices=["promote", "bump", "minor", "patch"],
-        help="the type of release to tag",
+        nargs="?",
```

Comment:
[Documentation for the cryptically-named `nargs` argument](https://docs.python.org/3/library/argparse.html#nargs).